lost-sample/src/main/java/com/example/lost/SettingsApiActivity.java

```diff
@@ -0,0 +1,205 @@
+  TextView networkUsable;
+  TextView locationPresent;
+  TextView locationUsable;
+  TextView blePresent;
+  TextView bleUsable;
+
+  Status requestStatus;
+
+  PendingResult<LocationSettingsResult> result;
+
+  @Override protected void onCreate(@Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_settings_api);
+
+    checkLocationSettings = (Button) findViewById(R.id.check_location_settings);
+    checkLocationSettings.setOnClickListener(new View.OnClickListener() {
+      @Override public void onClick(View v) {
+        checkLocationSettings();
+      }
+    });
+    resolveLocationSettings = (Button) findViewById(R.id.resolve_location_settings);
+    resolveLocationSettings.setEnabled(false);
+    resolveLocationSettings.setOnClickListener(new View.OnClickListener() {
+      @Override public void onClick(View v) {
+        resolveLocationSettings();
+      }
+    });
+    cancelBtn = (Button) findViewById(R.id.cancel);
+    cancelBtn.setOnClickListener(new View.OnClickListener() {
+      @Override public void onClick(View v) {
+        if (result != null) {
+          result.cancel();
+        }
+      }
+    });
+
+    gpsPresent = (TextView) findViewById(R.id.gps_present);
+    gpsUsable = (TextView) findViewById(R.id.gps_usable);
+    networkPresent = (TextView) findViewById(R.id.network_present);
+    networkUsable = (TextView) findViewById(R.id.network_usable);
+    locationPresent = (TextView) findViewById(R.id.location_present);
+    locationUsable = (TextView) findViewById(R.id.location_usable);
+    blePresent = (TextView) findViewById(R.id.ble_present);
+    bleUsable = (TextView) findViewById(R.id.ble_usable);
+  }
+
+  private void checkLocationSettings() {
+    ArrayList<LocationRequest> requests = new ArrayList<>();
+    LocationRequest highAccuracy = LocationRequest.create().setPriority(
+        LocationRequest.PRIORITY_HIGH_ACCURACY); //gps + wifi
+    LocationRequest balancedPowerAccuracy = LocationRequest.create().setPriority(
+        LocationRequest.PRIORITY_BALANCED_POWER_ACCURACY); // wifi
+    LocationRequest lowPower = LocationRequest.create().setPriority(
+        LocationRequest.PRIORITY_LOW_POWER); // wifi
+    LocationRequest noPower = LocationRequest.create().setPriority(
+        LocationRequest.PRIORITY_NO_POWER); // gps or wifi or none
+
+    requests.add(highAccuracy);
+    //requests.add(balancedPowerAccuracy);
+    //requests.add(lowPower);
+    //requests.add(noPower);
+
+    boolean needBle = true;
+    LocationSettingsRequest request = new LocationSettingsRequest.Builder()
+        .addAllLocationRequests(requests)
+        .setNeedBle(needBle)
+        .build();
+    LostApiClient apiClient = new LostApiClient.Builder(this).build();
+    apiClient.connect();
+    result = LocationServices.SettingsApi.checkLocationSettings(apiClient, request);
+
+    //result.setResultCallback(new ResultCallback<LocationSettingsResult>() {
```

Comment:
I get why this is here to show the different ways to invoke the settings API but I don't think we want to commit so much commented out code to the sample app as it makes it harder to follow.

If you think this is helpful to keep in the example what about refactoring this code into a private method then then just commenting out the method call?